comment/comment.go

```diff
@@ -89,6 +90,24 @@ func (m *GormCommentRepository) Save(ctx context.Context, comment *Comment) (*Co
 	return comment, nil
 }
 
+// Delete a single comment
+func (m *GormCommentRepository) Delete(ctx context.Context, comment *Comment) error {
+	c := Comment{}
+	tx := m.db.Where("id=?", comment.ID).First(&c)
+	if tx.RecordNotFound() {
```

Comment:
@kwk Am I sure? No, but on SQL level it does. If Gorm doesn't then that is a bug in Gorm.